quarkus-jnosql-mongodb/src/main/java/org/jnosql/demoee/DevelopersResource.java

```diff
@@ -22,7 +22,7 @@ public List<Developer> listAll() {
         return template.select(Developer.class).result();
     }
 
-    @Path("/byName")
+    @Path("/findByName")
     @GET
     public List<Developer> findByName(@QueryParam("name") String name) {
```

Comment:
What happens if "name" is empty? Should we list all instead?

Reply:
Yeah... it was intended to be a sample code, but you're right... I'm fixing it. Thanks, @otaviojava